examples/src/main/java/org/apache/spark/examples/sql/streaming/JavaStructuredNetworkWordCount.java

```diff
@@ -61,7 +61,8 @@ public static void main(String[] args) throws Exception {
       .load();
 
     // Split the lines into words
-    Dataset<String> words = lines.as(Encoders.STRING()).flatMap(new FlatMapFunction<String, String>() {
+    Dataset<String> words = lines.as(Encoders.STRING())
+      .flatMap(new FlatMapFunction<String, String>() {
       @Override
```

Comment:
FYI, there is style guides here, https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide and here https://github.com/databricks/scala-style-guide